lib/pages/chat/chat.dart

```diff
@@ -560,7 +569,20 @@ class ChatController extends State<Chat>
     }
   }
 
-  void _handleDownloadFileForPreview({required Event event}) async {
+  void _handlePreviewWeb({required Event event}) async {
+    if (!event.hasAttachment) {
+      Fluttertoast.showToast(msg: L10n.of(context)!.errorPreviewingFile);
+      return;
```

Comment:
change to to`Snackbar`

Reply:
nice, I forgot to change fluttertoasts tyvm